manticore/core/plugin.py

```diff
@@ -0,0 +1,158 @@
+import logging
+from ..utils.helpers import issymbolic
+logger = logging.getLogger('MANTICORE')
+
+class Plugin(object):
+    event_names = [
+        'did_enqueue_state','did_execute_instruction',
```

Comment:
We should eventually mark all the events a class can generate as a class constant and aggregate them here, otherwise we're duplicating these.

Reply:
We could explore the Eventful.__subclassess__() idea